inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -154,16 +154,18 @@ def _match_with_invenio_matcher(obj, eng):
     return _match_with_invenio_matcher
 
 
-def was_already_harvested(record):
-    """Return True if the record was already harvested.
+def was_already_harvested_on_legacy(record):
+    """Return True if the record was already harvested on Legacy
```

Comment:
The name of the function misleads me, I'd prefer something like `is_harvested_on_legacy` or `is_being_harvested_on_legacy` to point out that the record might not be yet harvested there, just that is legacy the one that should harvest it eventually.